packages/dashboard-core-plugins/src/PandasPlugin.tsx

```diff
@@ -66,6 +97,7 @@ export const PandasPlugin = (props: PandasPluginProps): JSX.Element => {
 
   useListener(layout.eventHub, PandasEvent.OPEN, handleOpen);
   useListener(layout.eventHub, PandasEvent.CLOSE, handleClose);
```

Comment:
Do we still need `PandasEvent.OPEN` and `PandasEvent.CLOSE`?

Reply:
Nope, this can be removed.